modules/nf-core/megahit/main.nf

```diff
@@ -33,11 +34,15 @@ process MEGAHIT {
         -t ${task.cpus} \\
         --out-prefix ${prefix}
 
+    export kmax=\$(grep "k-max reset to" megahit_out/*log | rev | cut -f 2 -d ' ' | rev)
+    megahit_toolkit contig2fastg \$kmax megahit_out/*.fa > megahit_out/${prefix}.contigs.fastg
```

Comment:
would it work on empty .fa? 
I thought maybe we need a check that assembly.fa exists or not empty...